packages/components/addon/components/hds/table/index.hbs

```diff
@@ -0,0 +1,42 @@
+<table class={{this.classNames}} ...attributes>
+  {{#if this.isSortableTable}}
+    <caption class="sr-only" aria-live="polite">{{@caption}} {{this.sortedMessageText}}</caption>
+  {{else}}
+    <caption class="sr-only">{{@caption}}</caption>
+  {{/if}}
```

Comment:
Question/concern: Is the caption required? If it isn't, the caption would render as an empty tag with no value.

Reply:
That's correct; it could render as an empty component. In the case of the sortable table, it renders as empty if there's no caption and then updates the message once the sort has happened.

Reply:
Ok thanks. I would prefer to not render an empty element but I guess if you need it there to display a message for the sortable table after it's sorted I understand.